### PR TITLE
CLOUDP-261435: Broken GOV tests - change AWS region

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:

* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if there is one).
* [ ] Update docs/release-notes/release-notes-template.md if your changes should be included in the release notes for the next release.

### CODE REVIEW

File: pkg/controller/connectionsecret/connectionsecrets.go [hunk 1]
```diff
@@ -215,6 +214,7 @@ func IsCloudGovDomain(ctx *workflow.Context) bool {
 		"cloud-qa.mongodbgov.com",
 	}
 
+	fmt.Printf("DBG: DOMAIN: %s", ctx.Client.BaseURL.Host)
```
Comment (s-urbaniak) [inline pkg/controller/connectionsecret/connectionsecrets.go:217]: let's remove this

File: pkg/controller/connectionsecret/connectionsecrets.go [hunk 2]
```diff
@@ -196,13 +196,12 @@ func FillPrivateConnStrings(connStrings *mongodbatlas.ConnectionStrings, data *C
 }
 
 func GetAllServerless(ctx *workflow.Context, projectID string) ([]*mongodbatlas.Cluster, error) {
+	if IsCloudGovDomain(ctx) {
+		return make([]*mongodbatlas.Cluster, 0), nil
+	}
```
Comment (s-urbaniak) [inline pkg/controller/connectionsecret/connectionsecrets.go:201]: is this a separate issue? I think this solves a separate issue, namely that serverless is not supported at all in cloud gov. Hence, I would like to see this in a separate contribution with a separate ticket.

Comment (igor-karpukhin) [reply pkg/controller/connectionsecret/connectionsecrets.go:201]: I wouldn't say it is a fix, more like an optimisation. There is no need to call "GetServerlessDeployments" if we're running on cloud-gov.

File: pkg/controller/connectionsecret/connectionsecrets.go (hunk 1 above)
Comment (igor-karpukhin) [reply pkg/controller/connectionsecret/connectionsecrets.go:217]: Sure. This is just to see if we getting in there in the CI/CD

**Pull request closed without merging**